### ISSUE TITLE
Review Post Import

### ISSUE THREAD

**Issue by artlogic** [arsmentis/wdet-conversion#44]
Please review the following:

- [ ] Did the post title, content and excerpt pull over properly (remember to turn on the excerpt view in the WordPress editor)
- [ ] Is the post authorship information correct, including multi-author posts?
- [ ] Are shows, series, and topics linked correctly?
- [ ] Are tags linked correctly?
- [ ] Is the featured image linked correctly?
- [ ] Does the publish time match between sites?

This list isn't exhaustive - there may be more things to check. I would encourage additions to be made to this list.

I am aware that #7 is still an issue, and I'm waiting on #38 to be resolved before tackling it. I don't expect it'll take long to resolve.

**Comment by artlogic** [comment]
I see that - I'm not sure what's going on there - I'm going to look into it.

**Comment by metastange** [comment]
Issues on a sampling of posts. Seeing consistent issues with publish _time_ (but not date), thumbnails, and pulling in multiple authors. I know we've discussed thumbnail and multiple authors before, so apologies if this is duplicative. Categories, titles, excerpts and written content of posts (not the inline images, many of which are missing) were all correct in what I reviewed.

Old: https://wdet.org/admin/wdet/post/91548/?_changelist_filters=q%3Dmichigan%2Bhouse%2Badopts
New: https://wdetcms.wdet.org/wp-admin/post.php?post=11181&action=edit
-The "Author" on the backend is showing as Dorothy Hernandez (the user who created this post, but Rick Pluta is the bylined author), but it's populating correctly (showing Rick Pluta as the author) on the front end. How can we assign authors in the WP backend?
-Incorrect thumbnail

old: https://wdet.org/admin/wdet/post/91520/?_changelist_filters=q%3Drep.%2Bpeter%2Bmeijer
new: https://wdetcms.wdet.org/wp-admin/post.php?post=11153&action=edit
-publish time is different (this is fine, but want to make sure i'm flagging discrepancies)
-The "Author" on the backend is showing as Jake Neher (the user who created this post, but Detroit Today is the bylined author), but it's populating correctly (showing Detroit Today as the author) on the front end. How can we assign authors in the WP backend?

old: https://wdet.org/admin/wdet/post/91460/?_changelist_filters=q%3Dfreep%2Bfilm%2Bfestival%2Bhighlights
new: https://wdetcms.wdet.org/wp-admin/post.php?post=11096&action=edit
-publish time is different (this is fine, but want to make sure i'm flagging discrepancies)

old: https://wdet.org/admin/wdet/post/91482/?_changelist_filters=q%3Dstate%2Bbudget%2Bdeal%2Bleaves%2Bbillions%2Bunspent
new: https://wdetcms.wdet.org/wp-admin/post.php?post=11116&action=edit
-Incorrect thumbnail
-only pulled in one author (Cheyna Roth), not both (should include Jake Neher as well)
-publish time is different

old: https://wdet.org/admin/wdet/post/91444/?_changelist_filters=q%3Ddetroit%2Bpolice%2Bofficer%2Bsuspended
new: https://wdetcms.wdet.org/wp-admin/post.php?post=11080&action=edit
-publish time inaccurate
-only has one author (missing Russ)

**Comment by artlogic** [comment]
Author issues you are seeing appear to be because someone (@wdet30?) turned off the authors plugin for all posts, defaulting back to the default WordPress single author infrastructure. I would get that feature turned back on and then see if the the problems with authors persist.

I'm looking into the other issues currently.

**Comment by wdet30** [comment]
James please turn them back on.  I'm AFK.


Thanks Tony
________________________________

**Comment by artlogic** [comment]
I have re-enabled multiple authors for posts. It looks like many of the spot checked posts have lost authorship information and I'm not sure why (this is where an audit log would be handy). On my system, these posts have the correct authors. We should definitely re-check these on re-import. Let me know if it's not clear how to modify author information on posts now that the plugin is re-enabled.

**Comment by artlogic** [comment]
I've looked into the thumbnail issue and it looks as if the import process is VERY sensitive to the admin being used during the import - even just to view things. I'm going to open a new issue to add some extra safeguards to make sure this doesn't happen, but we'll want to check against the next import.

**Comment by artlogic** [comment]
I've reviewed the post time issue and I've discovered a bug in time zone handling in the import. I will open a new issue to fix it.

**Comment by metastange** [comment]
> I have re-enabled multiple authors for posts. It looks like many of the spot checked posts have lost authorship information and I'm not sure why (this is where an audit log would be handy). On my system, these posts have the correct authors. We should definitely re-check these on re-import. Let me know if it's not clear how to modify author information on posts now that the plugin is re-enabled.

Thank! The author modification process is clear now that the plugin has been reenabled.

**Comment by metastange** [comment]
Closing this following the latest import.
